rmklaus12/spec/toy_robot/cli_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+
+RSpec.describe ToyRobot::CLI do
+  subject { ToyRobot::CLI.new }
+
+  context 'load_commands' do
+    it 'loads commands from a file' do
+      path = File.dirname(__FILE__) + '/../fixtures/commands.txt'
+      commands = subject.load_commands(path)
+      expect(commands).to eq([
+          [:place, 0, 0, 'NORTH'],
+          [:move],
+          [:turn_right],
+          [:turn_left],
+          [:report]
+                             ])
```

Comment:
spacing is a bit off, and it looks like the indentation here has 4 spaces whereas other lines have 2...